src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,236 @@
+     */
+    public static boolean isDeadline(String type) {
+        return type.equals(KEYWORD_DEADLINE);
+    }
+    /**
+     * Checking if the type of command is of type event.
+     * 
+     * @param type The inputted user's type.
+     * @return Boolean value of true if user's inputted type is event, false otherwise.
+     */
+    public static boolean isEvent(String type) {
+        return type.equals(KEYWORD_EVENT);
+    }
+
+    /**
+     * Checking if the type of command is of type done.
+     * 
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is done, false otherwise.
+     */
+    private static boolean isValidDone(String type) {
+        return type.equals(KEYWORD_DONE);
+    }
+
+    /**
+     * Checking if the type of command is of type bye.
+     * 
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is bye, false otherwise.
+     */
+    private static boolean isEnd(String type) {
+        return type.equals(KEYWORD_BYE);
+    }
+
+    /**
+     * Checking if the type of command is of type list.
+     * 
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is list, false otherwise.
+     */
+    private static boolean isList(String type) {
+        return type.equals(KEYWORD_LIST);
+    }
+
+    /**
+     * Checking if the type of command is of type task(deadline or todo or event).
+     * 
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is task, false otherwise.
+     */
+    private static boolean isTask(String type) {
+        return isDeadline(type) || isToDo(type) || isEvent(type);
+    }
+
+    /**
+     * Checking if the type of command is of type delete.
+     * 
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is delete, false otherwise.
+     */
+    private static boolean isDelete(String type) {
+        return type.equals(KEYWORD_DELETE);
+    }
+
+    /**
+     * Checking if the type of command is of type find.
+     *
+     * @param type The inputted user's type.
+     * @return boolean value of true if user's inputted type is find, false otherwise.
+     */
+    private static boolean isFind(String type) {
+        return type.equals(KEYWORD_FIND);
+    }
+    /**
+     * Formats the user's input timing into a LocalDateTime format.
+     * 
+     * @param s The inputted user's timing.
+     * @return A LocalDateTime object that contains the information of the timing.
+     * @throws InvalidFormatDateException If the inputted user's timing is not of the correct format,
+     * YYYY-MM-DD HHMM or YYYY-MM-DD HMMM, an exception will be thrown to notify the user.
+     */
+    public static LocalDateTime formatDateTime(String s) throws InvalidFormatDateException {
```

Comment:
You can be more explicit for parameter names so its easier for the code reviewers 👍

Reply:
Thanks